src/linker/configuration.py

```diff
@@ -12,21 +12,17 @@ class Config:
 
     def __init__(
         self,
-        pipeline_specification: Optional[Path],
-        input_data: Optional[Path],
+        pipeline_specification: Path,
```

Comment:
Why are these no longer optional?

Reply:
It was a decision made along time ago - pipeline specification and input data are now click "required options". I had just forgotten to update the type hints